contracts/VAIController.sol

```diff
@@ -158,21 +178,23 @@ contract VAIController is VAIControllerStorageG2, VAIControllerErrorReporter, Ex
      * @return (uint, uint) An error code (0=success, otherwise a failure, see ErrorReporter.sol), and the actual repayment amount.
      */
     function repayVAIFresh(address payer, address borrower, uint repayAmount) internal returns (uint, uint) {
+        accrueVAIInterest();
```

Comment:
`Fresh` usually means the interest has been accrued already. Maybe move interest accrual to `repayVAI`?